### PR TITLE
Sentry integration

### LINKED ISSUE #70: Integrate Sentry (or something else more appropriate) for error reporting
As @eu9ene and I discussed, we should have better error logging and reporting being submitted to us.

### PR DESCRIPTION
**Pull request by eu9ene**
Sentry wrappings are required to capture errors in async methods and event listeners that are not propagated to the main onerror handler.

fixes #70

### CODE REVIEW

File: extension/manifest.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
   "background": {
     "persistent": true,
     "scripts": [
+      "sentry.js",
```
Comment (andrenatal) [inline extension/manifest.json:28]: can we add this file to `model\static\errorReporting`?

File: extension/manifest.json [hunk 2]
```diff
@@ -44,6 +45,7 @@
         "<all_urls>"
       ],
       "js": [
+        "sentry.js",
```
Comment (andrenatal) [inline extension/manifest.json:48]: same as above

File: extension/settings/prod.js [hunk 3]
```diff
@@ -2,5 +2,7 @@
 const settings = {
     logTelemetry: false,
     uploadTelemetry: true,
-    sendDebugPing: false
+    sendDebugPing: false,
+    sentryDsn: "https://9c324392ba52495d8e2de947420950a5@o1069899.ingest.sentry.io/6293991",
```
Comment (andrenatal) [inline extension/settings/prod.js:6]: just a layman's question: what's the mechanism used by sentry to prevent spam here?

**Review by andrenatal — CHANGES_REQUESTED** [review]
lgtm. let's just address the files' placement

File: extension/settings/prod.js (hunk 3 above)
Comment (eu9ene) [reply extension/settings/prod.js:6]: https://forum.sentry.io/t/how-does-sentry-prevent-spammers/8188 there is no protection on Sentry side, but I set a limit of 30 events per minute and we can also configure server-side filtering of events if we see something weird there

File: extension/manifest.json (hunk 1 above)
Comment (eu9ene) [reply extension/manifest.json]: done

**Review by andrenatal — APPROVED** [review]

**Pull request merged**